packages/types/src/primitive/StorageKey.ts

```diff
@@ -70,6 +70,14 @@ export default class StorageKey extends Bytes {
 
       assert(isFunction(fn), 'Expected function input for key construction');
 
+      if (fn.meta.type.isDoubleMap) {
```

Comment:
For safety, rather use `if (fn.meta && fn.meta.type. isDoubleMap)` - it is possible that it is just a function passed in since it can come from anywhere. (The `fn.method` and `fn.section` below doesn't have those issues since they will just be `undefined`)

Reply:
Ah. cool! Thank you @jacogr ! I will change it now.